tests/test_put_sentence.rs

```diff
@@ -182,7 +182,12 @@ fn test_put_sentence_structure_returns_200() {
         tests_commons::SERVICE_URL,
         sentence_uuid,
     );
-    let modified_structure = "<sentence>This is one sentence</sentence>";
+    let modified_structure = r#"
+        <subject>This</subject>
```

Comment:
You're missing a root element so it should not be accepted + space are significant so you can't indent like in a programming language